internal/regex/regex.go

```diff
@@ -15,12 +16,16 @@ type Matcher interface {
 }
 
 type matcher struct {
-	goDaddyKey, goDaddySecret, duckDNSToken, namecheapPassword, dreamhostKey, cloudflareKey, cloudflareUserServiceKey,
-	dnsOMaticUsername, dnsOMaticPassword *regexp.Regexp
+	gandiKey, goDaddyKey, goDaddySecret, duckDNSToken, namecheapPassword, dreamhostKey, cloudflareKey,
+	cloudflareUserServiceKey, dnsOMaticUsername, dnsOMaticPassword *regexp.Regexp
```

Comment:
Just being a low git diff maniac 😄 

```suggestion
	goDaddyKey, goDaddySecret, duckDNSToken, namecheapPassword, dreamhostKey, cloudflareKey,
	cloudflareUserServiceKey, dnsOMaticUsername, dnsOMaticPassword, gandiKey *regexp.Regexp
```